src/xrCore/xr_shortcut.h

```diff
@@ -17,21 +17,22 @@ struct XRCORE_API xr_shortcut
         {
             u8 key;
             Flags8 ext;
-        };
+        } ext_key;
```

Comment:
Why?

Reply:
It seems to be a unnamed union, and adding a constructor provides some compiler errors. It resolves it.